tasks/actorstate/actorstatechange.go

```diff
@@ -182,35 +181,15 @@ func (p *ActorStateChangeProcessor) processTipSet(ctx context.Context, node lens
 			return xerrors.Errorf("parse address: %w", err)
 		}
 
-		_, err = node.StateGetActor(ctx, addr, pts.Key())
-		if err != nil {
-			if strings.Contains(err.Error(), "actor not found") {
-				ll.Debugw("actor not found", "addr", str)
-				// TODO consider tracking deleted actors
-				continue
-			}
-			return xerrors.Errorf("get actor: %w", err)
-		}
-
-		_, err = node.StateGetActor(ctx, addr, pts.Parents())
-		if err != nil {
-			if strings.Contains(err.Error(), "actor not found") {
-				ll.Debugw("parent actor not found", "addr", str)
-				// TODO consider tracking deleted actors
-				continue
-			}
-			return xerrors.Errorf("get actor parent: %w", err)
-		}
-
```

Comment:
This lookup will always fail for the case when a message that creates an actor exists in `pts` -- meaning the actor state exists in `ts`'s stateroot -- since it contains the result of message evaluation from pts.

Reply:
I _think_ we are not always guaranteed to see an `actor not found` error here since some lens's will run the VM and the VM may not return `actor not found`. When this happens we don't persist any actor state changes for a tipset, meaning we will be missing data.

Reply:
I have removed these calls because they are no longer necessary give the below change to model data.